### PR TITLE
Release 7

### PR DESCRIPTION
**Pull request by tejohnso**
@ahmedalsudani @fjvallarino please review

### CODE REVIEW

**Comment by ahmedalsudani** [comment]
@tejohnso do we need to change latest Player version as well? Isn't it  what the previous version of installer checked to detect whether it's on latest?

**Comment by fjvallarino** [comment]
:+1:

File: versions-config/electron-remote-components-win-32.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "ForceStable": false,
-  "LatestRolloutPercent": 0,
-  "InstallerElectronVersion": "2016.03.22.02.99",
+  "LatestRolloutPercent": 2,
+  "InstallerElectronVersion": "2016.03.29.15.56",
   "InstallerElectronURL": "http://install-versions.risevision.com/first-bundle-upgrade-win-32.tar.gz",
```
Comment (ahmedalsudani) [inline versions-config/electron-remote-components-win-32.json]: @tejohnso we'll need to update the first bundle for win-32

Comment (tejohnso) [inline versions-config/electron-remote-components-win-32.json]: They've all been updated.  There's a new cache that had to go in.

**Pull request merged**